### DISCUSSION TITLE
Testing on Raspberry Pi 3

### DISCUSSION THREAD

**Discussion by paulocoutinhox** [mikke89/RmlUi#195] [Show and tell]
Hi,

To help share this project i made tests on my Raspberry Pi 3:
https://youtu.be/cMHx-o6I0kA

Some images:

![IMG_0284](https://user-images.githubusercontent.com/395096/120872806-1ab7d480-c576-11eb-9ce4-f2bcfada5592.jpg)
![IMG_0281](https://user-images.githubusercontent.com/395096/120872807-1c819800-c576-11eb-9b87-646d0a81e6de.jpg)
![IMG_0279](https://user-images.githubusercontent.com/395096/120872808-1d1a2e80-c576-11eb-897b-52dff8bae317.jpg)
![IMG_0276](https://user-images.githubusercontent.com/395096/120872810-1e4b5b80-c576-11eb-9a1c-ab4a6a103b6c.jpg)
![IMG_0275](https://user-images.githubusercontent.com/395096/120872812-1ee3f200-c576-11eb-93f0-20e1a26fdd8e.jpg)
![IMG_0274](https://user-images.githubusercontent.com/395096/120872813-1f7c8880-c576-11eb-8d7b-736ae7333bbe.jpg)
![IMG_0272](https://user-images.githubusercontent.com/395096/120872814-20151f00-c576-11eb-9baa-5d03fa3c8e1e.jpg)
![IMG_0270](https://user-images.githubusercontent.com/395096/120872816-20adb580-c576-11eb-8e13-fb596b31ddb0.jpg)
![IMG_0269](https://user-images.githubusercontent.com/395096/120872818-20adb580-c576-11eb-92c3-09081cb26408.jpg)
![IMG_0267](https://user-images.githubusercontent.com/395096/120872819-21464c00-c576-11eb-9443-3bbe0cc3923b.jpg)
![IMG_0265](https://user-images.githubusercontent.com/395096/120872820-21dee280-c576-11eb-8fd1-f6a4d12829d1.jpg)
![IMG_0263](https://user-images.githubusercontent.com/395096/120872821-21dee280-c576-11eb-8a42-6c48a112f4b4.jpg)
![IMG_0261](https://user-images.githubusercontent.com/395096/120872822-22777900-c576-11eb-8abb-3bc73ccb3c1d.jpg)

**Comment by mikke89** [comment]
This is awesome, thanks! Very delighted to see it run there, I can see the library being a nice fit for some touch screen interface projects.
